diff --git a/src/lib/pages/signout.tsx b/src/lib/pages/signout.tsx
index ab841d96798960af99548954cdb8fe2ee5dc82f0..9997d375fed2e2539e0ca6e5f7ab9ff465a75e21 100644
--- a/src/lib/pages/signout.tsx
+++ b/src/lib/pages/signout.tsx
@@ -37,7 +37,7 @@ export default function SignoutPage(props: SignoutProps) {
         {theme?.logo && <img src={theme.logo} alt="Logo" className="logo" />}
         <h1>Signout</h1>
         <p>Are you sure you want to sign out?</p>
-        <form action={`${url}/signout`} method="POST">
+        <form action={`${url}`} method="POST">
           <input type="hidden" name="csrfToken" value={csrfToken} />
           <button id="submitButton" type="submit">
             Sign out